File: Modeling/app/src/main/java/pisarev/com/modeling/interfaces/MainMvp.java
package pisarev.com.modeling.interfaces;

public interface MainMvp {

    interface ViewMvp {
        void showProgram(String program);
    }

    interface PresenterMainMvp {
        void openProgram(String path);

        void saveAll(String program, String path);
    }
}

File: Modeling/app/src/main/java/pisarev/com/modeling/interfaces/MainMvp.kt
package pisarev.com.modeling.interfaces

interface MainMvp {
    interface ViewMvp {
        fun showProgram(program: String?)
    }

    interface PresenterMainMvp {
        fun openProgram(path: String?)
        fun saveAll(program: String?, path: String?)
    }
}